src/express-route-cross-connection/setup.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from codecs import open
+from setuptools import setup, find_packages
+
+VERSION = "0.0.7"
```

Comment:
You probably want `0.1.0` if this will be added to the index.
If this will be a work in progress kind of thing, `0.0.1`